src/main/java/com/google/sps/data/ScheduledInterview.java

```diff
@@ -0,0 +1,44 @@
+ // Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+/** Represents a scheduled interview. Contains the timerange of the interview and the emails of the attendees. */
+public class ScheduledInterview {
+  private TimeRange when;
```

Comment:
Question: l see TimeRange doesn't exist yet, so this won't compile. Put a TODO here?